src/ci/pgo.sh

```diff
@@ -130,3 +130,7 @@ rm -r ./build/$PGO_HOST/llvm ./build/$PGO_HOST/lld
 $@ \
     --rust-profile-use=${RUSTC_PROFILE_MERGED_FILE} \
     --llvm-profile-use=${LLVM_PROFILE_MERGED_FILE}
+
+echo "Rustc binary size"
+ls -la ./build/$PGO_HOST/stage2/bin
+ls -la ./build/$PGO_HOST/stage2/lib
```

Comment:
do you intend to keep these? or are they just for debugging?

Reply:
Well they are for debugging (currently the PR is quite experimental, that's why it's still a draft). But I think that it would be nice to have this information in the CI log, otherwise we have to download the artifacts to find out the exact binary size (the CI log currently shows just the aggregate size per artifact archive).